File: rest/incoming-phone-numbers/list-get-example-3/list-get-example-3.py
# Download the Python helper library from twilio.com/docs/python/install
from twilio.rest import TwilioRestClient

# Your Account Sid and Auth Token from twilio.com/user/account
account_sid = "ACdc5f132a3c49700934481addd5ce1659"
auth_token  = "{{ auth_token }}"
client = TwilioRestClient(account_sid, auth_token)

# A list of number objects with the properties described above
numbers = client.phone_numbers.list(phone_number="867")

File: rest/incoming-phone-numbers/list-get-example-3/list-get-example-3.js
// Download the Node helper library from twilio.com/docs/node/install
// These vars are your accountSid and authToken from twilio.com/user/account
var accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
var authToken = "{{ auth_token }}";
var client = require('twilio')(accountSid, authToken);

client.incomingPhoneNumbers.list({ 
  phoneNumber: "867" 
  }, function(err, data) {
    data.numbers.forEach(function(number) {
        console.log(number.voiceUrl);
    });
});